migrations/20230627120414-create-look-up-table.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const lookUpTableName = 'look_up';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable(lookUpTableName, {
+      id: {
+        type: Sequelize.STRING,
+        primaryKey: true,
+        allowNull: false,
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      tokenized_name: {
+        type: Sequelize.DataTypes.TSVECTOR,
+        allowNull: false,
+      },
+      item_id: {
+        type: Sequelize.STRING(36),
```

Comment:
The user_id has to be `Sequelize.DataTypes.STRING` because the users' table was created like that, but in the case of the files/folders, the type is the proper one: `Sequelize.DataTypes.UUID`, could you adjust it?